packages/loader/container-utils/src/error.ts

```diff
@@ -113,27 +113,30 @@ export class DataProcessingError extends LoggingError implements IErrorBase, IFl
      * Conditionally coerce the throwable input into a DataProcessingError.
      * @param originalError - Throwable input to be converted.
      * @param message - Sequenced message (op) to include info about via telemetry props
-     * @param errorCodeIfNone - pascaleCased code identifying the call site, used if originalError has no error code.
+     * @param dataProcessingCodepath - which codepath failed while processing data.
      * @returns Either a new DataProcessingError, or (if wrapping is deemed unnecessary) the given error
      */
     static wrapIfUnrecognized(
         originalError: any,
-        errorCodeIfNone: string,
+        dataProcessingCodepath: string,
         message?: ISequencedDocumentMessage,
     ): IFluidErrorBase {
         const newErrorFn = (errMsg: string) => {
-            const dpe = new DataProcessingError(errMsg, errorCodeIfNone);
-            dpe.addTelemetryProperties({ untrustedOrigin: 1}); // To match normalizeError
+            const dpe = new DataProcessingError(errMsg, "none" /* fluidErrorCode */);
```

Comment:
i think errorCodeIfNone still applies in this case. the caller is giving a value, but you are setting it to none. seem strange

Reply:
maybe i get it. i will still prefer undefined over none